code/datums/mutations/speech.dm

```diff
@@ -106,30 +106,7 @@
 	text_lose_indication = "<span class='danger'>Your mind feels more clear.</span>"
 
 /datum/mutation/human/unintelligible/say_mod(message)
-	if(message)
-		var/prefix=copytext(message,1,2)
-		if(prefix == ";")
-			message = copytext(message,2)
-		else if(prefix in list(":","#"))
-			prefix += copytext(message,2,3)
-			message = copytext(message,3)
-		else
-			prefix=""
-
-		var/list/words = splittext(message," ")
-		var/list/rearranged = list()
-		for(var/i=1;i<=words.len;i++)
-			var/cword = pick(words)
-			words.Remove(cword)
-			var/suffix = copytext(cword,length(cword)-1,length(cword))
-			while(length(cword)>0 && suffix in list(".",",",";","!",":","?"))
-				cword  = copytext(cword,1              ,length(cword)-1)
-				suffix = copytext(cword,length(cword)-1,length(cword)  )
-			if(length(cword))
-				rearranged += cword
-		message ="[prefix][jointext(rearranged," ")]"
-	return message
-
+	return unintelligize(message)
```

Comment:
Instead of adding a new var on /mob/living, instead have the genetic trait and the brain trauma both add TRAIT_UNINTELLIGIBLE_SPEECH (from different sources).

Reply:
Actually, I'll just open the PR.